src/components/Help/HelpBanner/index.tsx

```diff
@@ -62,7 +62,7 @@ export const HelpBanner = ({ topic }: Props) => {
   return (
     <Card
       title={
-        <span>
+        <span className={css('vf-card')}>
```

Comment:
I would not use `vf-card` here. I'm not really a fan of the drop shadow on this title. it gives the impression of a card inside a card.